### ISSUE TITLE
Addition of a Mild tag.

### ISSUE THREAD

**Issue by Tat011** [captaincobbs/ESG-Mod#1490]
While Hot/Cold has Temperate as a middle ground, Sterile/Fertile does not, this results in planets that aren't Sterile/Fertile being straight up worse than other planets so I'm proposing adding the mild tag to fill this void. (Which already exists in that other balance mod I'm pretty sure)
There are some things to consider about this imlpementation tho:
-it needs to be integrated so presumably some buildings/pops need to be changed to favor this tag. (which ones?)
-would gas giants have this tag? (probably not but it's worth considring)

**Comment by captaincobbs** [comment]
Mild implies you're living in the atmosphere, I'm fine with adding this tag but keep that in mind. The Gas is a replacement for planet fertility.

Also, would this be associated with its own FIDS?

One last thing, could you make a list of buildings that would be affected and what their bonuses would be?

**Comment by Tat011** [comment]
Proposed pop changes:
-Make Haroshem work with Mild rather than Sterile (Fits their lore/theme better but we need to see how this affects Cravers)
-Make Xirmisala work with Mild rather than Fertile (would this be a nerf or buff to Nakalim?)
-Make Kalgeros work with Mild rather than Fertile
-To compensate for the loss of 2 fertile pop reduce Amoeba from +5 food to +4 and give them the unused +1 influence on fertile.

Proposed Improvement changes:
This one im not so sure on...
-We could give the resource deposits specialization a bonus on mild like almost all the other specialization get and give both deposit improvements bonuses on mild.
Improvements potentially affected: Strategic resource specialization, Miners Union, Expanded Mines, Craft Guilds, Mechsuit Quarter.
Pops potentially affected: Harmony, Gnashast, Galvran.

-We could give Influence an association with mild since they also have a non specific specialization (non unique planets would still make 0 influence natively tho).
Improvements potentially affected: Influence specialization, National Museum, Body Language Institute, Intergalactic Lectures.
Pops potentially affected: Imperials, Illo.

-Approval could be moved from being sort of associated with fertile exept not really (Fertile is also already taken by food) to mild (would require changing planets not insignificantly), which could be paired with adding an approval specialization.
Improvements potentially affected: Infinite Supermarkets, Colonial Rights, Wellbeing Foundation, High Serenity Programm
Pops potentially affected: Kalgeros, Xirmisala.

-We could be spicy and give military improvements an association with mild (probably a bad idea).
Improvements potentially affected: The ones that give ship EXP.
Pops potentially affected: Yuusho, Tikanan.

-We could make it so mild is associated with trade which could be paired with adding a trade income specialization.
Improvements potentially affected: Solar Security Operations Base, Xhiyun Cartel, Borer's Guild Borse.

**Comment by wakystuf** [comment]
This will be more of a lift than some other tweaks we've discussed, but if someone else is undertaking it then it's fine with me. A few thoughts:
-The changes to Xirmisala and Kalgeros seem fine. I'd probably increase the overall Amoeba number if we're going to add a condition (maybe +3 food unconditional, and then +3 on fertile?)
-I think you make a good point about needing to be careful with the Haroshems due to the Cravers; moving them to mild is kind of a buff to Craver early game (since Mild is easier to colonize than Sterile), which is not what the world needs.
-Associating mild with Influence probably makes the most sense (resources and military would be really susceptible to being broken, and approval should follow the terraforming tree for both lore and gameplay flow reasons). Trade could also work, though probably less good than influence
-If we do pick influence, then I agree with how you described it (non-unique planets still have 0 native influence, but the buildings--and I agree with the ones you listed--will have an added synergy). Probably roughly a +50% synergy? So If it's +X today, then it'd be +X unconditional (same as before) and +0.5X on Mild?
-Probably also add Culture Center to the list, though I would probably only create the synergy with the per pop portion (I believe there is also a fixed or per SD bonus that I'd leave as-is)

**Comment by Tat011** [comment]
Since aparently i can't change the Haroshems and and i assume changing the Epistis from sterile to Mild will also be rejected for the same reason im kinda out of pops to change, i guess Kalgeros can be changed, idk any ideas?

**Comment by wakystuf** [comment]
I thought we already talked about Kalgeros, right? I thought your idea was good on that one.

**Comment by Tat011** [comment]
i mentioned Haroshem, Xirmisala (which turns out got changed to on happy and the wiki never got updated), Kalgeros and Illo for if we went the influence route and since all of these minus Kalgeros are mayor faction fodder then i guess Kalgeros will be the only one to change.

**Comment by wakystuf** [comment]
Illo I think might be OK. They are secondary with Hissho, right?

**Comment by Tat011** [comment]
yes, currenty they have +1 influence and +2 influence on hot... for some reason

**Comment by captaincobbs** [comment]
I did some thinking and discussing with Frost, before you get too far on this, what would Mild's association even be? I don't want to add this just to add this.

Right now we have two Axis, as you well know:
- Temperature (Science - Dust - Industry)
- Sterility (High Population, Less Base FIDSI - High Base FIDSI, Less Base Population)

What would we conceivably associate Mild with, aside as a lame middle ground for some bonuses to target.

**Comment by Tat011** [comment]
i gave a bunch of options and waky said influence (while still keeping the raw production as 0) was the one that makes the most sense and i agree.

**Comment by captaincobbs** [comment]
Could have Temperate as Influence, Mild as Dust, it would match vanilla better

**Comment by Tat011** [comment]
That'd require changing a lot of stuff and idk why it'd be better exactly.

**Comment by captaincobbs** [comment]
It would match the vanilla allocation of dust, and influence was originally planned to be associated with Temperate but they killed most of the influence buildings later on.

**Comment by Tat011** [comment]
Ok im pretty sure the only thing left now are modfying the improvements.

**Comment by Tat011** [comment]
Improvement changes:
Culture Center: +50 per system level > +50 per Mild planet
National Museum: +2 per pop on happy/ecstatic > +2 per pop on Mild
Added Mild conditional to influence specialization
Body Language Institute: +15%, +3 per pop > +15%, +2 per pop, +2 per pop on Mild
Intergalactic Lectures: +10% per peace, +20% per alliance, +3 per pop > +10% per system level, +2 per pop, + 2 per pop on Mild
aaand that's it, this is finished

**Comment by captaincobbs** [comment]
Not a big fan of associating Mild with Influence, doesn't feel like it makes sense. imo, Temperate could be Influence, and Mild could be Dust.

**Comment by Tat011** [comment]
I don't really see it. But the thing is, even if i did, this would require changing all the planet fidsi as well as all the dust on temperate stuff on top of the influence stuff too. All that just to swap inf and dust? Who's even gonna do that?

**Comment by captaincobbs** [comment]
I mean, Mild basically already is Dust, no changes needed really. We don't even need to include Inf, but having Dust on Mild is the only thing I could reasonably see it doing, Influence doesn't really feel like it needs to be attached to a planet type aside from Unique, if anything, Unique would be a more interesting thing to key for.

**Comment by Tat011** [comment]
You haven't made any argument besides that you personally dont like it for some unknown reason and yes, this would require changes.
Swamp and Gas Temperate would need to change, the dust/pacifist on temperate pops would need to change and so would the dust on temperate buildings, oh and dont forget the influence stuff would need to change too, all of this so that dust can be asociated with middle plants instead of middle weather, which is not a significant difference and actually makes less lore sense since temperate weather facilitates trade while there's no justification why temperate plants cause more dust.
There is also a justification with why influence on mild which is that with sterile people dont go outside their habitats much cuz of the hostile conditions and with fertile they also wont do that (though to a lesser extent) cuz of all the plantlife/wildlife/possibly dangerous microbes.
And, again, who'd do this exactly? cuz it certainly seems like a second dust rework is being imposed on me.

**Comment by captaincobbs** [comment]
I don't think this change would add anything without more work than would be needed, and Influence being associated with Mild breaks the base game design that we've been adhering to (Which, generally I try to stick to as closely as possible).

Influence is generally associated with infrastructure/monuments (Unique Planets, System Development), not to the planet's gameplay type.

You make a good point about swapping the proposed associations, but I still strongly disagree that Mild should be associated with anything. We can leave it in for other modders or rare use cases, such as the Population Capacity improvements that currently do not target Mild planets at all, but I just don't see us re-associating Influence to Mild will do anything aside from make us need to rework how Influence in general is given. Best case, as is I think we'd need to rework this later and give it serious attention, which I don't think anyone really has the motivation for doing so.

If you want to make the Influence buildings have a bonus on Unique Planets or something, that's completely fine and doesn't conflict with the vanilla game's design, but aside from that I really don't see a reason to re-associate the influence buildings.

**Comment by Tat011** [comment]
-I don't see how changing inf to be on mild is more of a "break in game design" than changing dust from sterile to temperate is, if anything it's less since this doesn't change planet raw FIDSI or minor pop FIDSI (though i might change Kalgeros to be on mild, idk for sure).
-Influence is associated with culture, all inf improvements are about boosting your cultural output and unique planets fit in by the mere fact that they're unique.
-If Mild isn't associated with anything then it'd be a waste of a feature, also it'd be the only tag without any kind of association.
-The changes to inf buildings have already been done, we can just test it to see what the effect is like on a real game.
-You can't associate influence with unique because you can't ensure unique spawning will be fair and uniques are already a strong source of influence.

Also can we get opinions of people who aren't us 2? @wakystuf @Kil10naire @Arpharnator @Zingybird @Provolonage

**Comment by captaincobbs** [comment]
For what it's worth I dislike having changed Dust from Sterile to Temperate, I wouldn't change it back since it solves several issues quite nicely. 
And no, I meant the source of Influence, not what it literally represented.
Yes, I don't think the Mild tag should be added in general.

**Comment by Tat011** [comment]
Ok im willing to compromize so here's a proposal that swaps mild and temperate:
Planets:
Swamp: Change values from 2/2/2/16 to 5/5/5/5
Gas Temperate: takes the values from warm: 0/10/10/10
Gas Warm: 0/0/15/15 (Industry/Science)

Pops:
Amblyr: Temperate > Mild
Kalgeros: Fertile > Temperate

Buildings:
Swapping influence/dust specialization tags
Culture Center/National Museum/Body Language Institute/Intergalactic Lectures: Mild > Temperate

Borer's Guild Bourse: Temperate > Mild
Adaptative Taxation systems: Temperate > Mild (alternatively we could do Fertile > Mild)

I think this covers everything.

**Comment by Arpharnator** [comment]
I think that this is less and less necessary.
There is no need for a mild tag, no need for that kind of change and neither is there a need for a compromise in the middle.

**Comment by Tat011** [comment]
Yeah there is, i think pretty much everyone can agree that all the in between fertile/sterile planets are just straight up worse and that is because they don't have a sterile/fertile tag, tags give planets a lot more viability which is why it was necesary to give Gas Warm the Temperate tag.
Currently the meta is still to terraform to Desert/Artic or to Fertile (no one is terraforming to Monsoon or Steppes or whatever) and while this is very likely still gonna be the case with these changes it's at the very least gonna make those planets less bad.

Also it gives something modders/custom faction makers can use.

**Comment by captaincobbs** [comment]
Them being bad is more just a consequence of the bad planet system in general, I don't see how adding a mild tag would do anything other than add a spectrum where there doesn't really need to be one. I had a long conversation with Frost about this earlier this morning and we couldn't come up with a solution that
a) Was worse than what we had now
b) Didn't involve ripping out the entire design and replacing it

**Comment by Provolonage** [comment]
If you want to jsut throw a +1 infl on mild to each buildings I dont really care as long as the general functionality remains unchanged. What Cobbs mean I think is that planetary features don't dictate infl gen in any version of the sandbox, planets only have FIDS yields baseline and tech tree improvements aren't effected by planet type either. The only infl thing that works off any planet feature is the quest buildings that gives +8 per temperate. I like the idea of the Mild tag alot but I think its place in the current sandbox is to give leeway and flexibility to the generally really shitty sterile and temperature dependent improvements. Currently the best planets for every faction except RB and cold sophons are fertile temp and I don't think anyone's goal right now is to change that.

**Comment by captaincobbs** [comment]
No followup, closing
